projects/adrv9009/s10soc/system_qsys.tcl

```diff
@@ -14,3 +15,11 @@ if [info exists ad_project_dir] {
 } else {
   source ../common/adrv9009_qsys.tcl
 }
+
+#system ID
+set_instance_parameter_value axi_sysid_0 {ROM_ADDR_BITS} {9}
+set_instance_parameter_value rom_sys_0 {ROM_ADDR_BITS} {9}
+
+set_instance_parameter_value rom_sys_0 {PATH_TO_FILE} "$mem_init_sys_file_path/mem_init_sys.txt"
+
+sysid_gen_sys_init_file
```

Comment:
Please add an empty line at the end of the file.

Reply:
Done.